playwright-e2e/tests/rgp/dsm-family-enrollment.spec.ts

```diff
@@ -551,7 +551,7 @@ test.describe.serial('DSM Family Enrollment Handling', () => {
 
     //Setup new family member
     const grandfather = new FamilyMemberTab(page, FamilyMember.MATERNAL_GRANDFATHER);
-    await grandfather.setRandomRelationshipID();
```

Comment:
I cannot remember why I initially made this randomized - but now with a test expecting a particular relationship id, changed it to using the set one